### PR TITLE
Duct blaster measurements

### PR DESCRIPTION
**Pull request by bpark1327**
Address NREL/OpenStudio-HEScore#280

## Pull Request Description

Adds ability to accept CFM25 duct leakage to outside value (supply + return)

## Checklist

PR Author: Check these when they're done. Not all may apply. ~~strikethrough~~ and check any that do not apply. 

PR Reviewer: Verify each has been completed.

- [x] Code changes (must work)
- [x] Test exercising your feature or bug fix. Check the coverage report in the build artifacts.
- [x] All other unit tests passing
- [x] Update translation docs

### CODE REVIEW

**Comment by shorowit** [comment]
@bpark1327 Could you pull in the latest `master` and resolve the conflict? Thanks.

**Comment by bpark1327** [comment]
> @bpark1327 Could you pull in the latest `master` and resolve the conflict? Thanks.

It's done. This branch is up to date.

File: hescorehpxml/base.py [hunk 1]
```diff
@@ -560,6 +560,18 @@ def get_hvac_distribution(self, hvacd_el, bldg):
                        '(h:DuctLeakageMeasurement/h:LeakinessObservedVisualInspection="connections sealed w mastic") ' +
                        'or (ancestor::h:HVACDistribution/h:HVACDistributionImprovement/h:DuctSystemSealed="true")')
 
+        # Duct leakage to outside
+        leakage_to_outside = \
+            self.xpath(airdist_el,
+                       'h:DuctLeakageMeasurement/h:DuctLeakage[h:TotalOrToOutside="to outside"]/h:Value/text()')
+
+        if leakage_to_outside:
```
Comment (nmerket) [inline hescorehpxml/base.py:568]: Looks like preference is given to the duct blaster measurement. I think that's the right choice.

**Review by nmerket — CHANGES_REQUESTED** [review]
Reminder to self: I need to make a release of hescore-hpxml before this or any other breaking changes get merged. I think there were some things @charliepnnl wanted to get out there sooner than the January release.

**Review by nmerket — APPROVED** [review]

**Pull request merged**